crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -22,12 +26,13 @@ pub struct AzureConverter {
     pub(crate) clock: Box<dyn Clock>,
     pub(crate) size_threshold: SizeThreshold,
     pub(crate) mapper_config: MapperConfig,
+    pub mqtt_schema: MqttSchema,
 }
 
 impl AzureConverter {
     pub fn new(add_timestamp: bool, clock: Box<dyn Clock>) -> Self {
         let mapper_config = MapperConfig {
-            out_topic: Topic::new_unchecked("az/messages/events/"),
+            out_topic: "az/messages/events/".to_string(),
```

Comment:
This could have been just a const. I'd also drop the trailing `/` and when subtopics need to be built out of this prefix, let them add the `/` before the subtopic. I find something like `{0}/{source}/m` to be far more explicit and less error-probe than `{0}{source}/m`.

Reply:
There is also an error topic. So, I thought its better to keep them together.